UnityProject/Assets/Scripts/Items/ItemAttributesV2.cs

```diff
@@ -182,6 +182,8 @@ public IEnumerable<string> ServerAttackVerbs
 		[SerializeField]
 		private ItemsSprites itemSprites;
 
+		public bool IsFakeItem = false;
```

Comment:
I don't think this needs to be shown in the inspector.

`public bool IsFakeItem  {get; set;} = false;`